index.html

```diff
@@ -5,42 +5,105 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Leverage</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./css/main.css">
 </head>
 
 <body>
     <!-- header: start -->
-    <header id="main_header" class="container"><div class="row">HEADER</div></header>
+    <header id="main_header" class="container">
+        <div class="row">HEADER</div>
+    </header>
     <!-- header: end -->
     <!-- hero: start -->
-    <div id="hero" class="container bg-light-gray"><div class="row">HERO</div></div>
+    <div id="hero" class="container bg-light-gray">
+        <div class="row">HERO</div>
+    </div>
     <!-- hero: end -->
     <!-- features: start -->
-    <div id="features" class="container bg-dark-gray"><div class="row">FEATURES</div></div>
+    <div id="features" class="container bg-dark-gray">
+        <div class="row">FEATURES</div>
+    </div>
     <!-- features: end -->
     <!-- about_us: start -->
-    <div id="about_us" class="container bg-light-gray"><div class="row">ABOUT US</div></div>
+    <div id="about_us" class="container bg-light-gray">
+        <div class="row col-12 about">
+            <div class="row col-12 about-structure">
+                <h2>About us</h2>
+                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent diam lacus,
+                    dapibus
+                    sed imperdiet consectetur.</p>
+
+                <div class="block">
+                    <div class="block-segment">
+                        <div class="icon">
+                            <i class="fa fa-th-large" aria-hidden="true"></i>
+                        </div>
+                        <div class="content">
+                            <div class="content-title">
+                                Design
+                            </div>
+                            <div class="content-description">
+                                Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+                            </div>
+                        </div>
+                    </div>
+
+                    <div class="block-segment">
+                        <div class="icon">
+                            <i class="fa fa-free-code-camp" aria-hidden="true"></i>
+                        </div>
+                        <div class="content">
+                            <div class="content-title">
+                                Branding
+                            </div>
+                            <div class="content-description">
+                                Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="row col-12 about-image">
```

Comment:
originale foto veikia kaip nuoroda, kurią paspaudus atsidaro full size foto.
Šitą reiktų arba padaryti iškart, arba prie render issue prirašyti kaip užduotį, kad foto yra nuoroda

Reply:
Pritariu, kad reiktu sukurti kaip naują issues'ą nuotraukos atvaizdavimui paspaudus ant jos.